app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.kt

```diff
@@ -0,0 +1,20 @@
+package ar.com.wolox.android.example.ui.login
+
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+
+class LoginPresenter constructor() : BasePresenter<LoginView>() {
+
+    fun onClickLogin(fistName: String, lastName: String) {
+        view?.dummyFunction()
+    }
+
+    fun onClickSingup() {
+        view?.dummyFunction2()
```

Comment:
Don't use numbers like part of names, you should use literals, so, if you going to use dummyFunction2 should rename to dummyFunctionTwo